packages/matter-node-ble.js/src/ble/BlenoServer.ts

```diff
@@ -0,0 +1,334 @@
+/**
```

Comment:
I would name this file BtpServer.ts

Reply:
I change to BlenoBleServer becasue it has nothing to do with BTP beside being the bridge.